static/app/components/actions/menuItemActionLink.tsx

```diff
@@ -31,12 +31,6 @@ const StyledActionLink = styled(ActionLink)`
   }
 `;
 
-const MenuItemActionLink = styled(MenuItemActionLinkBase)`
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
-
-  &:last-child {
-    border-bottom: none;
-  }
-`;
+const MenuItemActionLink = styled(MenuItemActionLinkBase)``;
```

Comment:
Is this still needed?

Reply:
It could be if a component was using it as a styled component selector, let me try and check.

Reply:
Ok I think it's fine to kill